[2024-02-10T03:42:57.645+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 1_First_job.call_my_name scheduled__2022-08-21T00:00:00+00:00 [queued]>
[2024-02-10T03:42:57.660+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 1_First_job.call_my_name scheduled__2022-08-21T00:00:00+00:00 [queued]>
[2024-02-10T03:42:57.661+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-02-10T03:42:57.681+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): call_my_name> on 2022-08-21 00:00:00+00:00
[2024-02-10T03:42:57.691+0000] {standard_task_runner.py:60} INFO - Started process 11148 to run task
[2024-02-10T03:42:57.695+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', '1_First_job', 'call_my_name', 'scheduled__2022-08-21T00:00:00+00:00', '--job-id', '254', '--raw', '--subdir', 'DAGS_FOLDER/1_Firstjob.py', '--cfg-path', '/tmp/tmpsud31_s1']
[2024-02-10T03:42:57.698+0000] {standard_task_runner.py:88} INFO - Job 254: Subtask call_my_name
[2024-02-10T03:42:57.765+0000] {task_command.py:423} INFO - Running <TaskInstance: 1_First_job.call_my_name scheduled__2022-08-21T00:00:00+00:00 [running]> on host fb7af9b6b8e4
[2024-02-10T03:42:57.883+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='snowadmin' AIRFLOW_CTX_DAG_ID='1_First_job' AIRFLOW_CTX_TASK_ID='call_my_name' AIRFLOW_CTX_EXECUTION_DATE='2022-08-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-08-21T00:00:00+00:00'
[2024-02-10T03:42:57.885+0000] {logging_mixin.py:188} INFO - Hi my name is :Python
[2024-02-10T03:42:57.886+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-10T03:42:57.898+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=1_First_job, task_id=call_my_name, execution_date=20220821T000000, start_date=20240210T034257, end_date=20240210T034257
[2024-02-10T03:42:57.947+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-10T03:42:58.068+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
